qemu/tests/cfg/numa_opts.cfg

```diff
@@ -15,62 +15,46 @@
       numa_nodes = 0
     - nodes.1:
       smp_override = 6
-      mem_override = 256
+      mem_override = 4096
```

Comment:
For windows guests, only datacenter support hotplug_mem, so I have no concern about 4G mem for hotplug_mem case.
But how about numa_opts case, windows desktop will run this job, right? 
Windows desktop can only recognize about 3G, is it meaningful to set guest to 4G mem?

Reply:
@lijinlijin Yes, windows desktop guest would run this case as well.  I think it is ok to set 4G here since the case only checks the numa topology from monitor not in guest.